libpod/network/network.go

```diff
@@ -194,8 +194,10 @@ func removeNetwork(config *config.Config, name string) error {
 				return errors.Wrapf(err, "failed to get live network names")
 			}
 			if util.StringInSlice(interfaceName, liveNetworkNames) {
-				if err := RemoveInterface(interfaceName); err != nil {
-					return errors.Wrapf(err, "failed to delete the network interface %q", interfaceName)
+				err = RemoveInterface(interfaceName)
```

Comment:
Nit: I find the `if err = ...; err != nil` syntax more idiomatic.

Reply:
ok fixed